docs/includes/usage-examples/DeleteOneTest.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Http\Controllers;
+
+use App\Models\Movie;
+use MongoDB\Laravel\Tests\TestCase;
+
+class DeleteOneTest extends TestCase
+{
+    /**
+     * @runInSeparateProcess
+     * @preserveGlobalState disabled
+     */
+    public function testDeleteOne(): void
+    {
+        require_once __DIR__ . '/Movie.php';
+
+        Movie::truncate();
+        Movie::insert([
+            [
+                'title' => 'Capote',
+                'runtime' => 114,
+            ],
+        ]);
+
+        // begin-delete-one
+        $deleted = Movie::where('title', 'Capote')
+            ->orderBy('_id')
+            ->first()
+            ->delete();
```

Comment:
Did you know you can also write a query to remove the document without fetching it:

```php
$deleted = Movie::where('title', 'Capote')
    ->delete();
```

`$deleted` is then an int with the number of deleted documents.

Reply:
Since this is a delete one usage example (rather than [delete multiple](https://jira.mongodb.org/browse/DOCSP-35973)), I used `first()` to make sure only one document was retrieved. If there's a better way to delete only one document, let me know!

Reply:
If `limit(1)` is called, the limit is applied. There is a special handling for that: https://github.com/mongodb/laravel-mongodb/blob/295d12914d20b4de467a625c3bff8ea4ac77548b/src/Query/Builder.php#L780-L788